src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,55 @@
+package duke.task;
+
+public abstract class Task {
+    private final String name;
+    private boolean done;
```

Comment:
I think that according to the coding standard booleans should be named more obviously so done can be isDone.

Reply:
I have another method called isDone, so I think I will leave it as that (although can also rename to getIsDone but done is a private variable so I would prefer having isDone name on the public method instead)